slides/ds101-worskhop.mdx

```diff
@@ -116,3 +118,53 @@ function Code({ children, language, inline = false }) {
         <a href="https://component-driven.io">https://component-driven.io</a>
     }
 />
+
+---
+
+# Timetable
+
+<Timetable
+    start="9:30"
+    items={[
+	    {
+		    title: "Introduction",
+		    length: "0:30"
+	    },
+        {
+            title: "Ice breaker: baking a toast",
+            length: "0:30"
+        },
+        {
+            title: "Introduction to the design systems",
+            length: "0:45"
+        },
+        {
+            title: "Prepare! Naming exercise",
+            length: "0:30"
+        },
+        {
+            title: "break",
+            length: "0:15"
+        },
+        {
+            title: "Start! UI inventory",
+            length: "01:00"
+        },
+        {
+            title: "Execute! Parts of design system",
+            length: "0:45"
+        },
+        {
+            title: "break",
+            length: "0:15"
+        },
+        {
+            title: "Maintain! Pattern Journey",
+            length: "2:00"
+        },
+        {
+            title: "Recap",
+            length: "0:30"
+        }
+    ]}
+/>
```

Comment:
Let's not focus on not important things for now. Timetable isn't moving us closer to the completion.

Reply:
1. It's already done. We loose no time merging it.
2. It is helpful to understand how long every part of the workshop takes and how long the whole thing gets. In my previous experience, it has always been a hassle. Usually people tend to plan a lot while timing is limited. We have to shape the exercises keeping this in mind. Summing up times is a nice reminder.

Reply:
We went this route with the ReasonConf website and we ended up spending 2 man-days fixing issues caused by users looking at the time table from different time zones. Maybe we won't get those issues but I'm really reluctant to merge something that has non-zero maintenance const and low value.

> Usually people tend to plan a lot while timing is limited. We have to shape the exercises keeping this in mind. Summing up times is a nice reminder.

I don't believe that having this data in JSON going to help us and it's easy enough to count the duration so I would rather not a technical solution to a simple problem.